pkg/windowsdesktop/sfx/Microsoft.WindowsDesktop.App.Ref.sfxproj

```diff
@@ -16,24 +16,18 @@
     -->
   <ItemGroup>
     <FrameworkListFileClass Include="Accessibility.dll" Profile="WindowsForms;WPF" />
-    <FrameworkListFileClass Include="Microsoft.Win32.Registry.AccessControl.dll" Profile="WindowsForms;WPF" />
```

Comment:
Microsoft.Win32.Registry.AccessControl isn't part of the `Microsoft.NETCore.App.Ref` targeting pack. Should you don't want to expose it in the windowsdesktop targeting pack anymore? That would be a breaking change.